src/Sylius/Bundle/AdminBundle/Resources/config/app/config.yml

```diff
@@ -906,28 +906,25 @@ sylius_ui:
             blocks:
                 before_header_legacy:
                     template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
-                    priority: 55
```

Comment:
I'm not sure how if we should change already defined priorities. We may mess up someones UI